webapp/.eslintrc.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
   },
   extends: [
     'standard',
-    'plugin:vue/recommended',
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-05-31T11:44:53Z" title="Friday, May 31st 2019, 1:44:53 pm +02:00">May 31, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -10,7 +10,7 @@ module.exports = {
   },
   extends: [
     'standard',
-    'plugin:vue/recommended',
```


@ulfgebhardt @Tirokk @mattwr18 @ogerly @Lulalaby I graduated here. I believe the two libraries are incomaptible since prettier supports vue linting.